Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -231,8 +252,10 @@ done
 [ -z "$HOST" ] && echo "(ERROR) empty host value." >&2 && exit 2
 [ -z "$PORT" ] && echo "(ERROR) empty port value." >&2 && exit 2
 [ -z "$GRAPH" ] && GRAPH=http://$HOST:$PORT/$GRAPH_PATH
-[ -z "$DELIMITER" ]  && DELIMITER=$'\0'
-[ "x$DELIMITER" = "xNOT SPECIFIED" ] && DELIMITER=$'\n'
+[ -z "$DELIMITER" ]  && DELIMITER=$'\4'
+[ "x$DELIMITER" = "xNOT SPECIFIED" ] && DELIMITER=$'\4'
```

Comment:
What it special about the `DELIMITER` variable being `"NOT SPECIFIED"` at this point?
I mean, in the earlier versions it used to mean "no batch mode". In this case the script had to handle messages one by one -- that is why  `\n` (hardcoded EOM marker) was used as the delimiter.

If now `"NOT SPECIFIED"` has no special effect, I believe it should be removed at all.

But I think we should keep the possibility to work in "no batch mode". Talking about this -- I think it would be more readable:
* to have some flag option to specify "batch mode on/off" (say, `--batch`);
* to use "empty delimiter" as force "no batch mode" (say, `--eob ''`);
* to use EOM as the delimiter for `read` command in "no bach mode".

It gives us something like this:
```
./uploadTTL.sh <...> --batch                # batch mode with default EOB
./uploadTTL.sh <...>                             # no batch mode
./uploadTTL.sh <...> --eob ''                # no batch mode
./uploadTTL.sh <...> --batch --eob ''   # no batch mode (empty EOB to cancel --batch option)
./uploadTTL.sh <...> --batch '\0'          # batch mode with custom EOB
./uploadTTL.sh <...> --eob '\0'             # batch mode with custom EOB (equivalent to above)
```